apps/trading/pages/markets/trade-grid.tsx

```diff
@@ -87,20 +89,18 @@ export const TradeMarketHeader = ({
             <span className={itemClassName}>Volume</span>
             <span data-testid="trading-volume" className={itemValueClassName}>
               {market.data && market.data.indicativeVolume !== '0'
-                ? market.data.indicativeVolume
+                ? formatNumber(market.data.indicativeVolume)
```

Comment:
chore (blocking): Probably want to format with market.positionDecimal places here.

Reply:
just updated